runtime/altair/src/xcm.rs

```diff
@@ -301,6 +279,9 @@ pub type LocationToAccountId = (
 	SiblingParachainConvertsVia<Sibling, AccountId>,
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
 	AccountId32Aliases<RelayNetwork, AccountId>,
+	// A wildcard MultiLocation to AccountId conversion for all the other MultiLocations
+	// within the same Relay network.
+	Account32Hash<RelayNetwork, AccountId>,
```

Comment:
This is needed to handle origins with a `DescendOrigin` composed multilocation. We added it a while back to `development` to support `Moonbase -> Development` Liquidity Pools messages being sent through XCM. Adding that support here now for Altair and also for Centrifuge.

Reply:
We need to use the compatible solution in the ecosystem for this. 

* https://github.com/moonbeam-foundation/polkadot/blob/d83bb6cc7d7c93ead2fd3cafce0e268fd3f6b9bc/xcm/xcm-builder/src/location_conversion.rs#L25C1-L68C2

I wanted to wait till we can actually use the one provided by parity but given that we need this now lets mimic the one from moonbeam. 

Tests can be copi pasta from the same repo.

Reply:
Thanks @mustermeiszer, TIL.

That's on Moonbeam's fork of Polkadot tho. To use that we would need to copy-paste it and share it across runtimes, which I guess is fine but runs the risk of getting out of sync.

> Tests can be copi pasta from the same repo.

what do you mean?

Reply:
> That's on Moonbeam's fork of Polkadot tho. To use that we would need to copy-paste it and share it across runtimes, which I guess is fine but runs the risk of getting out of sync.

We can switch to the maintained version once we are on `> polkadot-v0.9.43`.

Reply:
>> Tests can be copi pasta from the same repo.

>what do you mean?

We can copy paste the tests ^^

Reply:
The reason for this is that once users transfer money into their derived accounts we must support this. Moonbeam almost had this issue as Parity switch the approach on short notice...

Hope this explains the reasoning.

Reply:
Thanks @mustermeiszer, definitely :) it's done now 👍 The CI is acting up but it's looking good on my machine.